### PR TITLE
QuDynamics added to Projects.

### PR DESCRIPTION
**Pull request by amitjamadagni**
QuDynamics has been added to the projects list.

### CODE REVIEW

File: projects/index.md [hunk 1]
```diff
@@ -48,3 +48,11 @@ started by [jrevels](https://github.com/jrevels).
 A library for performing quantum mechanics using Dirac notation in Julia.
 Some of its functionalities have been merged into the [QuBase.jl](https://github.com/JuliaQuantum/QuBase.jl) project
 as a joint effort for building a basic quantum representation library for high-level JuliaQuantum projects.
+
+### [QuDynamics.jl](https://github.com/JuliaQuantum/QuDynamics.jl)
+
+A library for solving dynamical equations in quantum mechanics.
+It aims to provide a unified framework for solvers to solve various dynamical equations like
```
Comment (acroy) [inline projects/index.md]: "for solvers" is redundant.

File: projects/index.md [hunk 2]
```diff
@@ -48,3 +48,11 @@ started by [jrevels](https://github.com/jrevels).
...
+Schrodinger Equation, Liouville von Neumann Equation, Lindblad Master Equation. Various solvers
+have been integrated from packages like [ODE.jl](https://github.com/JuliaLang/ODE.jl), [ExpmV.jl](https://github.com/marcusps/ExpmV.jl),
+[Expokit.jl](https://github.com/acroy/Expokit.jl) using base features from [QuBase.jl](https://github.com/JuliaQuantum/QuBase.jl).
```
Comment (acroy) [inline projects/index.md]: Maybe instead of "using base features from" we could say "Currently, it depends on and makes use of features from"?

**Comment by i2000s** [comment]
Looks fine to me.

**Comment by acroy** [comment]
Indeed. I didn't see the update.

**Pull request merged**